comments/bad_sample.txt

```diff
@@ -0,0 +1,4 @@
+this is bad
```

Comment:
Could this be a `.csv` file? Just because we get a CSV exported from CloudMine.

Reply:
Added as CSV file